### ISSUE TITLE
http-api: Create documentation for API

### ISSUE THREAD

**Issue by sebastinez** [radicle-dev/radicle-client-services#145]
We should create a documentation for all the endpoints the API provides, to allow users and developers to build their own interfaces on top of the seed endpoints.

For a start I'll write down a Markdown file with the different routes, params and return values. Similar to this file https://github.com/Medium/medium-api-docs#2-authentication

**Comment by Davetbutler** [comment]
Hey @sebastinez. I know this is an old issue, but I was wondering if there was anywhere you made a start on the documentation? I am trying to piece a few things together by what is served on radicle's frontend, but if there was a central place for endpoint docs, or even just a list of endpoints, that would be great!

**Comment by sebastinez** [comment]
Hi @Davetbutler, yeah it really is an old issue, but important nevertheless.
I made some progress when I created the issue, you can find it in the `sebastinez/http-api-doc` branch https://github.com/radicle-dev/radicle-client-services/commits/sebastinez/http-api-doc

Will try to get back to it in the near future.

If you have any specific questions to endpoints feel free to ask them here or on our Discord server

**Comment by Davetbutler** [comment]
Thanks a lot for your response. I did not think to look for a branch that may have been created with initial work. It is really helpful.

My main question/interest is whether there are any "user" endpoints? Everything I have seen (now confirmed by your docs) is "project centric". 

For example, I would be really interested in being able to get data about [my "profile"](https://app.radicle.xyz/registrations/davetbutler.eth) (not that there is much on there atm). Or being able to call an endpoint that gives me all projects a user has committed to, for example.

Do these exist by any chance? 

Ps. [I did drop a message in the discord](https://app.radicle.xyz/registrations/davetbutler.eth), let me know if it was in the wrong channel. Of course it is totally fine for no-one to reply straight away - I know everyone is busy so finding time is hard!
